### PR TITLE
LPS-89013

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by ruben-pulido** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/headless-apio/headless-apio-demo/src/main/java/com/liferay/headless/apio/demo/internal/HeadlessDemo.java [hunk 1]
```diff
@@ -207,7 +211,8 @@ private FileEntry _createFileEntryInFolder(
 			serviceContext);
 	}
 
-	private Folder _createFolder(User user, Group group)
+	private Folder _createFolder(
+			User user, Group group, long folderId, String name)
```
Comment (ruben-pulido) [inline modules/apps/headless-apio/headless-apio-demo/src/main/java/com/liferay/headless/apio/demo/internal/HeadlessDemo.java]: Can we rename folderId to parentFolderId?

File: modules/apps/headless-apio/headless-apio-demo/src/main/java/com/liferay/headless/apio/demo/internal/HeadlessDemo.java [hunk 2]
```diff
@@ -195,6 +194,37 @@ private Group _createDemoGroup(Company company, User user)
 				user));
 	}
 
+	private FileEntry _createFileEntryInFolder(
```
Comment (ruben-pulido) [inline modules/apps/headless-apio/headless-apio-demo/src/main/java/com/liferay/headless/apio/demo/internal/HeadlessDemo.java]: I would call the method _createFileEntry

**Comment by ruben-pulido** [comment]
👏

**Comment by dgarciasarai** [comment]
Thanks! :)

**Pull request closed without merging**